modules/gitops-configure-rollout-route-traffic-using-openshift-service-mesh.adoc

```diff
@@ -443,49 +445,5 @@ $ oc argo rollouts get rollout rollouts-demo --watch -n <namespace> <1>
 <1> Specify the namespace where the `Rollout` resource is defined.
 --
 +
```

Comment:
Delete this `+`, that might be causing the issue

Reply:
Fixed.